lib/customer.rb

```diff
@@ -0,0 +1,42 @@
+require 'receipt'
+require 'menu'
+
+require 'order'
+
+
+class Customer
+
+  include Menu
+  include Receipt
+
+  def initialize
+    @order = Order.new
+  end
+
+  def list_of_menu_items
+    Menu.list
+  end
+
+  def print_list
```

Comment:
Nice extra feature to print the menu items to the user 👍

Reply:
thank you 👍